docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,99 @@
+# Rocket.Chat help: developing stronger questions
+
+## On this page
+* [**The 'how' and 'what'**](#the-'how'-and-'what')
+* [**How**](#how)
+* [**What**](#what)
+* [**The 'where'**](#the-'where')
+* [**The 'when'**](#the-'when')
+* [**Post the question**](#post-the-question)
+* [**Related pages**](#related-pages)
+<!-- ### End of "On this page" -->
 ---
-title: Get help in Rocket.Chat 
 
-slug: get-help-in-rocketchat
+The best support on Rocket.Chat comes when users ask detailed questions. They help the community respond quicker. An ideal user question looks like the example below:
 
-description: Describes how to obtain support in Rocket.Chat within the B.C. Government Rocket.Chat community and the different types of troubleshooting.
+> I've setup a new caddy webserver on OpenShift. I've created a route for it. But when I go to the url in my web browser I get an error page with "Application is not available". I confirmed the URL in the browser matches the route. I've tried with different browsers. I've checked the web server logs and the server is running with no errors. Here is my route:
 
-keywords: Rocket.chat, help in rocket.chat, support, help, community, developers, help, cluster issue, tool issue, stack issue, problem solving, 
 
-page_purpose: Helps developer find the best way to troubleshoot issues with clusters,  tools, or stacks
+> **route.yml YAML**
 
-audience: technical lead, openshift 101 students, openshift 201 students,  developers
+    kind: Route
+    apiVersion: route.openshift.io/v1
+    metadata:
+     name: my-web-server
+     namespace: abcd12-dev
+     labels:
+      app: my-web-server
+      group: my-project
+    spec:
+     host: my-web-server-abcd12-dev.apps.silver.devops.gov.bc.ca
+     to:
+      kind: service
+      name: my-web-server
+      weight: 100
+     port:
+      targetPort: web
+     tls:
+      termination: edge
+      insecureEdgeTerminalPolicy: Redirect
+     wildcardPolicy: None
 
-author: Alex Lloyd
+The user tells us the where, when, what, and how of an issue. We want to know what you expected, what happened, and in what environment, the problem happened. Before connecting with us, check [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) if you've done everything possible before posting.
```

Comment:
I'm wondering if it would be beneficial to return the definition of 'us/we'. I had it before then removed it. Using we/us in the context helps with creating a community vibe, but if that's not the intention, then I can replace pronouns with nouns. 

For the checklist, titling a link with 'this [noun]' doesn't align with accessibility, and writing wise, it's generally good practice to credit an author. A resolution could be a more descriptive link, does that work?